lib/stubs/object_ml.c

```diff
@@ -0,0 +1,135 @@
+#include "lablqml.h"
```

Comment:
I'm asking because it seems that we now are going to write `OCamlObject {}` in QML instead of `QtObject`. ANd I want to understand why.